### PR TITLE
Fix #7 and cancel selection while dragging

### LINKED ISSUE #7: Dragging handle throws exception in Firefox
ReferenceError: event is not defined

At line 332 of paper-slider.html.

### PR DESCRIPTION
**Pull request by blasten**

### CODE REVIEW

File: paper-slider.html [hunk 1]
```diff
@@ -330,15 +330,15 @@
...
+          this._trackStart(e);
           break;
         case 'track':
-          this._trackX(event);
+          this._trackX(e);
           break;
-        case 'start':
-          this._trackStart(event);
+        case 'end':
+          this._trackEnd(e);
```
Comment (cdata) [inline paper-slider.html]: `event` > `e` :bike: :house:

File: demo/index.html [hunk 2]
```diff
@@ -59,7 +59,7 @@
   </style>
 
 </head>
-<body>
+<body unresolved>
```
Comment (cdata) [inline demo/index.html:62]: Is this still a thing in Polymer > 0.5?

Comment (blasten) [inline demo/index.html:62]: yes

File: paper-slider.html [hunk 3]
```diff
@@ -350,14 +350,13 @@
       this._minx = - this._startx;
       this._maxx = this._w - this._startx;
       this.$.sliderKnob.classList.add('dragging');
+
       this._setDragging(true);
-      e.preventDefault();
```
Comment (cdata) [inline paper-slider.html:354]: Maybe the reason this doesn't work is that `e` is a synthetic `CustomEvent`, and not the actual MouseEvent / TouchEvent that caused the gesture?

File: paper-slider.html (hunk 1 above)
Comment (blasten) [inline paper-slider.html]: lol.

File: paper-slider.html (hunk 3 above)
Comment (blasten) [inline paper-slider.html:354]: I think so 😓

File: paper-slider.html (hunk 1 above)
Comment (cdata) [inline paper-slider.html]: Is it `event` or `error` `¯\_(ツ)_/¯`

File: paper-slider.html (hunk 3 above)
Comment (cdata) [inline paper-slider.html:354]: @azakus any thoughts here?

Comment (dfreedm) [inline paper-slider.html:354]: Yep, exactly that. You need `e.detail.sourceEvent.preventDefault()`

Comment (blasten) [inline paper-slider.html:354]: @azakus That's amazing!

**Pull request merged**